README.md

```diff
@@ -1 +1,212 @@
-# -hello-microvers
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up_2: please remove this logo if you want your own logo then put it here otherwise remove this image from here :slightly_smiling_face: